scripts/compendium.js

```diff
@@ -53,6 +51,5 @@ let observer = new IntersectionObserver(function(observerEntries, _) {
     }
 }, observerOptions);
 
-observer.observe(entries[0]);
```

Comment:
We still want this line so the left arrow gets removed when the user scrolls to the first entry (no more previous entries).

Reply:
Oh, sorry, I was unclear about what that did. I'll put it back.